src/styles/App.css

```diff
@@ -0,0 +1,18 @@
+.App {
```

Comment:
CSS classes should be written in kebab-case, no captials

Reply:
Corrected class name in App.js and App.css